scripts/buck_setup.sh

```diff
@@ -0,0 +1,27 @@
+cd third_party
```

Comment:
I suggest putting a `#!/bin/bash` to make sure bash is used on OSes where /bin/sh is a less capable shell.